src/DGmethods/test/Sphere/advection_sphere.jl

```diff
@@ -0,0 +1,290 @@
+    topl = StackedCubedSphereTopology(mpicomm,Nhorizontal,Rrange; boundary=(0,0))
+    (error, Δmass) = main(mpicomm, DFloat, topl, N, timeend, ArrayType, dt)
+end
+#}}} Run Script
+
+#{{{ Run Program
+using Test
+let
+    MPI.Initialized() || MPI.Init()
+    Sys.iswindows() || (isinteractive() && MPI.finalize_atexit())
+    mpicomm=MPI.COMM_WORLD
+
+    if MPI.Comm_rank(mpicomm) == 0
+        ll = uppercase(get(ENV, "JULIA_LOG_LEVEL", "INFO"))
+        loglevel = ll == "DEBUG" ? Logging.Debug :
+        ll == "WARN"  ? Logging.Warn  :
+        ll == "ERROR" ? Logging.Error : Logging.Info
+        global_logger(ConsoleLogger(stderr, loglevel))
+    else
+        global_logger(NullLogger())
+    end
+
+    #Perform Integration Testing for three different grid resolutions
+    if integration_testing
+        timeend = 1
+        numelem = (2, 2) #(Nhorizontal,Nvertical)
+        N = 4
+        ArrayType = Array
+        dt=1e-2*5 #stable dt for N=4 and Ne=5
+
+        expected_error = Array{Float64}(undef, 3) # h-refinement levels lvl
+        expected_error[1] = 1.5684355102513664e-01 #Ne=2
+        expected_error[2] = 8.8549044843092145e-03 #Ne=4
+        expected_error[3] = 2.2163001051913196e-04 #Ne=8
+        lvls = length(expected_error)
+
+        for DFloat in (Float64,) #Float32)
+            err = zeros(DFloat, lvls)
+            mass= zeros(DFloat, lvls)
+            for l = 1:lvls
+                Nhorizontal = 2^(l-1) * numelem[1]
+                Nvertical   = 2^(l-1) * numelem[2]
+                dt=dt/Nhorizontal
+                nsteps = ceil(Int64, timeend / dt)
+                dt = timeend / nsteps
+                @show(Nhorizontal,Nvertical,N,dt,nsteps)
+                (err[l], mass[l]) = run(mpicomm, Nhorizontal, Nvertical, N, timeend, DFloat, dt, ArrayType)
+                @test err[l] ≈ DFloat(expected_error[l])
+            end
+            @info begin
+                msg = ""
+                for l = 1:lvls-1
+                    rate = log2(err[l]) - log2(err[l+1])
+                    msg *= @sprintf("\n  rate for level %d = %e\n", l, rate)
+                end
+                msg
+            end
+        end
+    end
+    #Perform Integration Testing for three different grid resolutions
+
+    #=
+    #This snippet of code allows one to run just one instance/configuration. Before running this, Comment the Integration Testing block above
+    DFloat = Float64
+    N=4
+    ArrayType = Array
+    dt=1e-2*5 #stable dt for N=4 and Ne=5
+
+    timeend=1.0
+    Nhorizontal = 2 #number of horizontal elements per face of cubed-sphere grid
+    Nvertical = 2 #number of horizontal elements per face of cubed-sphere grid
+    dt=dt/Nhorizontal
+    nsteps = ceil(Int64, timeend / dt)
+    dt = timeend / nsteps
+    (error, Δmass) = run(mpicomm, Nhorizontal, Nvertical, N, timeend, DFloat, dt, ArrayType)
+    =#
+
+end #Test
+
+isinteractive() || MPI.Finalize()
+
+#nothing
+println("Done")
```

Comment:
I'd recommend removing this. Not needed.

Reply:
I took lines 287-289 directly from the Isentropic_Vortex_StandAlone example. Just trying to follow the same format.

Reply:
I just meant remove `println("done")`